src/main/java/org/spongepowered/api/event/entity/player/PlayerEvent.java

```diff
@@ -38,18 +38,6 @@
      *
      * @return The {@link Player} involved
      */
-    Player getPlayer();
-
-    @Override
-    Player getHuman();
-
-    @Override
-    Player getLiving();
-
     @Override
     Player getEntity();
-
-    @Override
-    Player getUser();
```

Comment:
Since `getUser()` is *required* for a `UserEvent`, I'd rather we have an override for `getUser()` returning `Player` in `PlayerEvent`.